scipy/optimize/_differentialevolution.py

```diff
@@ -64,10 +64,10 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
         A multiplier for setting the total population size.  The population has
         ``popsize * len(x)`` individuals.
     tol : float, optional
-        When the mean of the population energies, multiplied by tol,
-        divided by the standard deviation of the population energies
-        is greater than 1 the solving process terminates:
-        ``convergence = mean(pop) * tol / stdev(pop) > 1``
+         Relative tolerance for convergence, the solving stops when
```

Comment:
There appears to be a leading space before `R` which might be the cause refguide build failure.